### ISSUE TITLE
Ability to configure AYS job timeouts

### ISSUE THREAD

**Issue by rkhamis** [Jumpscale/jumpscale_core8#736]
Suggesting to do it in actor schema.

Something like:
```
timeout.<function name> = 1h,
```

See https://github.com/Jumpscale/jscockpit/issues/363

**Comment by zaibon** [comment]
In the 8.2.0 notation that would be in the config.yaml file:
```yaml
timeouts:
 - action_name: '1h'
```

**Comment by rkhamis** [comment]
@zaibon Is recurring notation also the same?

**Comment by zaibon** [comment]
recurring looks like :
```yaml
recurring:
   - action: action_name
     log: true
     period: 1m
```

**Comment by zaibon** [comment]
another way to do it would be to have an actions block and then add all differrents attribute of an action there. recurring, timeout, security . And if we don't specify detail for one action, then we have a default config.

```yaml
actions:
   - install:
      timeout: 10m
      recurring: 1h
   - monitor:
      timeout: 1m
      recurring: 10m
```

That maps pretty well with what we have in the blueprint.

**Comment by rkhamis** [comment]
In that case, let's stay consistent and do it like you suggested [here](https://github.com/Jumpscale/jumpscale_core8/issues/736#issuecomment-279371151)

**Comment by rkhamis** [comment]
according to @xmonader this has been implemented on 8.2.0

Will try cherry-picking his commits and see if works

**Comment by xmonader** [comment]
Implemented in this https://github.com/Jumpscale/jumpscale_core8/pull/820 also with AYS test service 
https://github.com/Jumpscale/ays_jumpscale8/commit/b07d413deb8199bccc2a6b0207f223813b2d8c6f
